velero-plugins/configmap/common.go

```diff
@@ -0,0 +1,18 @@
+package configmap
+
+import "strings"
+
+// Interested ConfigMap Suffixes
+// https://github.com/openshift/openshift-controller-manager/blob/aabcbc2cf5d944f64e6ebdbc4e0ce7f1b95bd127/pkg/build/buildutil/util.go#L44-L46
+const (
+	caConfigMapSuffix        = "ca"
+	globalCAConfigMapSuffix  = "global-ca"
```

Comment:
is this used anywhere?

Reply:
nah but just for consistency sake copied over as well can comment out or delete.

Reply:
```
L44-L46
```